templates/chiffee/redirect.html

```diff
@@ -0,0 +1,19 @@
+{% extends "chiffee/base.html" %}
+
+{% block head %}
+    <script>
+        function redirect() {
+            window.location.href = "/";
+        }
+
+        setTimeout(redirect, 3000);
+    </script>
+{% endblock %}
+
+{% block body %}
+    {% if done %}
+        <p>Erledigt! Umleiten...</p>
+    {% else %}
+        <p>Etwas ist schief gelaufen! Umleiten...</p>
+    {% endif %}
+{% endblock %}
```

Comment:
There should be a bigger difference between success and failure. People will not recognize this and assume everything went as planned.

So maybe make the background red for failures and give a better explanation on what went wrong.

Reply:
I added a giant error 404 in red. :P